docs/en/source/quickstart/quickstart_in_one_minute.ipynb

```diff
@@ -11,20 +11,13 @@
     "\n",
     "## Prerequisite\n",
     " - Ubuntu: `18.04`\n",
-    " - Python: `3.7.x`\n",
-    " - Flask: `1.1.2`\n",
-    " - MindSpore: `CPU-1.1.1`\n",
-    " - TinyMS: `0.1.0`\n",
-    " - numpy: `1.17.5`\n",
-    " - Pillow: `8.1.0`\n",
-    " - pip: `21.0.1`\n",
-    " - requests: `2.18.4`\n",
+    " - docker: `v18.06.1-ce`\n",
+    " \n",
+    " To use jupyter,you need to go to[Docker拉取tinyms镜像](https://tinyms.readthedocs.io/zh_CN/latest/quickstart/install.html)to obtain docker and pull tinyms0 3.0 mirror instructions\n",
```

Comment:
Do not leave Chinese characters, pay attention to spaces. The translation should be consistent with the Chinese documents.